content/en/BestPractices/_index.md

```diff
@@ -0,0 +1,33 @@
+---
+title: "Best Practices for Security"
+linkTitle: "Best Practices"
+weight: 600
+# The `toc_hide` entry hides the link from the menu. The content still available at <URL>/bestpractices/
+toc_hide: true
+description: >
+  Recommendations for developers focused on security.
+---
+
+{{% pageinfo %}}
+This document collects content from [our pentesters](https://cobalt.io/our-pentesters) to
+help you secure your systems.
+{{% /pageinfo %}}
+
+You want to start an [application security](../getting-started/glossary/#application-security-appsec)
+program. You've read through the 2021 version of the [OWASP Top 10](https://owasp.org/Top10/).
+We'll provide practical examples that you can use as patterns to secure your
+[assets](../getting-started/glossary/#assets). We'll revise our work to help you
```

Comment:
I am a little confused about this line. I feel like I am not understanding exactly what "work" it is that we are revising. Maybe I am reading it wrong?